webview/src/experiments/components/table/Row.tsx

```diff
@@ -100,53 +144,51 @@ const getSingleSelectMenuOptions = (
   id: string,
   isWorkspace: boolean,
   projectHasCheckpoints: boolean,
+  hasRunningExperiment: boolean,
   depth: number,
   queued?: boolean,
   starred?: boolean
 ) => {
-  const isNotCheckpoint = depth <= 1 || isWorkspace
-  const canApplyOrCreateBranch = queued || isWorkspace || depth <= 0
+  const hideApplyAndCreateBranch = queued || isWorkspace || depth <= 0
 
   const withId = (
     label: string,
     type: MessageFromWebviewType,
     hidden?: boolean,
     divider?: boolean
-  ) => experimentMenuOption(id, label, type, hidden, divider)
+  ) =>
+    experimentMenuOption(
+      id,
+      label,
+      type,
+      hidden || hasRunningExperiment,
+      divider
+    )
 
   return [
     withId(
       'Apply to Workspace',
       MessageFromWebviewType.APPLY_EXPERIMENT_TO_WORKSPACE,
-      canApplyOrCreateBranch
+      hideApplyAndCreateBranch
```

Comment:
[Q] The name would suggest the logic has been reversed here (but it looks like the same condition). Was the name wrong before and the logic correct or has something else happened?

Reply:
Yes, because the property is called `hidden`, so these are assertions that will hide the option when true. It was wrong before